src/beacon/app/dashboard/devices/page.tsx

```
@@ -21,34 +21,35 @@ import {
   Calendar,
   ChevronLeft,
   ChevronRight,
-  Layers,
-  Map
+  Map,
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)